src/api/client.ts

```diff
@@ -62,19 +65,14 @@ function responseErrorHandler(error: AxiosError) {
       break;
     case 'network-error':
     case 'timeout':
-      if (!isCancel(error)) {
-        // This happens all the time in mobile apps,
-        // so will be a lot of noise if we choose to report these
-        console.warn(errorType, error);
-        warnConfig(error.config);
+      if (!isCancel(error) && getBooleanConfigValue('enable_network_logging')) {
+        const errorMetadata = getAxiosErrorMetadata(error);
+        Bugsnag.notify(error, (event) => {
+          event.addMetadata('api', {...errorMetadata});
+        });
       }
       break;
   }
 
   return Promise.reject(error);
 }
-
-function warnConfig(config: AxiosRequestConfig) {
-  if (!config) return;
-  console.warn(`URL: ${config.baseURL}${config.url}`);
```

Comment:
URL is still warned in local development?

Reply:
Yeah that's handled by the override for `Bugsnag.notify` when `__DEV__` since it also includes metadata

Reply:
```bash
[Wed Feb 03 2021 14:41:00.931]  ERROR    [BUGSNAG] [Error: Request failed with status code 404] {
  "requestId": "5f3b7691-b720-449b-99d2-d882d5f33bd5",
  "requestUrl": "ticket/v1/search/zones2",
  "requestMessage": "Request failed with status code 404",
  "responseStatus": 404,
  "responseData": "{\"code\":404,\"message\":\"The current request is not defined by this API.\"}"
}
```